src/main/java/com/minecolonies/colony/buildings/AbstractBuilding.java

```diff
@@ -210,9 +223,11 @@ public static AbstractBuilding createFromNBT(Colony colony, NBTTagCompound compo
     /**
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)